docs/platforms/react-native/tracing/instrumentation/react-native-navigation.mdx

```diff
@@ -29,26 +29,16 @@ Sentry.init({
 You can configure the instrumentation by passing an options object to the constructor:
 
 ```javascript
-Sentry.reactNativeNavigationIntegration({
-  navigation: Navigation, // Navigation from `react-native-navigation`
-  routeChangeTimeoutMs: 1_000, // default: 1_000
-  enableTabsInstrumentation: true, // default: false
-  ignoreEmptyBackNavigationTransactions: true, // default: true
+const navigationIntegration = new Sentry.ReactNavigationInstrumentation({
```

Comment:
Hi @wsdt, thanks for contributing! 

I noticed in the code example there is an option for `enableTimeToInitialDisplay`, but in the list of options below it provides an explanation for `enableTabsInstrumentation` which is not in the code sample. Is this correct?